ui/pages/confirm-send-ether/confirm-send-ether.container.js

```diff
@@ -21,19 +29,13 @@ const mapDispatchToProps = (dispatch) => {
       const { id, txParams } = txData;
       const { from, gas: gasLimit, gasPrice, to, value: amount } = txParams;
 
-      dispatch(
-        updateSend({
```

Comment:
I removed this in favor of updating a few different states atomically. The updateSend method previously just used whatever it was sent as the new state which doesn't seem safe to me.